components/CaseStudiesHero.tsx

```diff
@@ -0,0 +1,30 @@
+const heroText = ["Case", "Studies"];
+
+function CaseStudiesHero() {
+  return (
+    <section className="flex flex-col items-center gap-5 bg-white800 px-6 py-12 text-center dark:bg-black300 lg:gap-[1.8rem]">
+      <h1 className="mobileHeading1 max-w-[21rem] dark:text-white800 sm:max-w-[41.125rem] lg:text-[4rem] lg:leading-[5.2rem] lg:tracking-[-0.04rem]">
+        Recent{" "}
+        {heroText.map((text) => (
+          <>
+            <span key={text} className="relative inline-block">
+              <span
+                className={`${
+                  text === "Case" && "sm:w-[15rem]"
+                } absolute top-[1.88rem] h-[0.8125rem] w-full shrink-0 bg-accentSecondary 
+                lg:top-[3.6rem] lg:h-[1.68963rem]`}
+              />
+              <span className="relative">{text}</span>
+            </span>{" "}
+          </>
```

Comment:
Here are a few things to review:
- Consider making this a reusable component.
- You don't need the fragment, A `<div>` would be more appropriate here. Also, you would have placed your key in the fragment, not the `<span>`
- I understand you don't have much data but it's still good practice to use a unique key when possible. 

```
// projectsData.ts
export const projects = [
  // ...
];

```

```
// Import it in your component
import { projects } from './projectsData';

```